locales/en/index.yml

```diff
@@ -59,6 +59,9 @@ global:
   badges:
     blocked: "Blocked"
     onGoing: "Ongoing"
+    allCaps:
+      failed:  "FAILED"
+      onGoing: "ONGOING"
```

Comment:
Why use a dedicated local key instead of transforming the text with `toUpper`?

Reply:
since the `failure` key is different, I felt that this would be the best way, also I'm not sure that the design's is the last version of the copy

Reply:
I think that should be `IOBadge` to change the case of the label, and not the label itself.
@dmnplb please correct me if I'm wrong.

Reply:
@forrest57 I agree with @mastro993. It's a visual attribute, so when we switch the component to capital case (as an example), we shouldn't be checking all string values through the app. We also managed this case in the new `Tag` and `Badge` components ([see reference](https://github.com/pagopa/io-app/blob/cd933b94598ae9f7b24050d5f61c599d114c6465/ts/components/core/Tag.tsx#L80))

Reply:
adressed in [b82d752](https://github.com/pagopa/io-app/pull/4710/commits/b82d7522602a7a9b7b102b4bfaa66fd5c28d14a1)